command/README.md

```diff
@@ -0,0 +1,87 @@
+<div align="center">
+  <h1>expo-github-action/command</h1>
+  <p>Run <a href="https://github.com/expo/expo-cli">Expo CLI</a> or <a href="https://github.com/expo/eas-cli">EAS CLI</a> command by a comment on the PR</p>
+  <p>
+    <a href="https://github.com/expo/expo-github-action/releases">
+      <img src="https://img.shields.io/github/v/release/expo/expo-github-action" alt="releases" />
+    </a>
+    <a href="https://github.com/expo/expo-github-action/actions">
+      <img src="https://img.shields.io/github/workflow/status/expo/expo-github-action/test" alt="builds" />
+    </a>
+    <a href="https://github.com/expo/expo-github-action/blob/main/LICENSE.md">
+      <img src="https://img.shields.io/github/license/expo/expo-github-action" alt="license" />
+    </a>
+  </p>
+  <p align="center">
+    <a href="#configuration-options"><b>Usage</b></a>
+    &nbsp;&nbsp;&mdash;&nbsp;&nbsp;
+    <a href="#example-workflows"><b>Examples</b></a>
+    &nbsp;&nbsp;&mdash;&nbsp;&nbsp;
+    <a href="#things-to-know"><b>Caveats</b></a>
+    &nbsp;&nbsp;&mdash;&nbsp;&nbsp;
+    <a href="https://github.com/expo/expo-github-action/blob/main/CHANGELOG.md"><b>Changelog</b></a>
+  </p>
+</div>
+
```

Comment:
```suggestion

## Experimental

**_This sub action is experimental and might change without notice. Use it at your own risk_**

```

Reply:
Let's add this! After that, you can merge it 😄

Reply:
Thank you, 😍